examples/games/game_menu.rs

```diff
@@ -2,7 +2,7 @@
 //! change some settings or quit. There is no actual game, it will just display the current
 //! settings for 5 seconds before going back to the menu.
 
-use bevy::prelude::*;
+use bevy::{prelude::*, text::DefaultFont};
```

Comment:
Should `DefaultFont` be part of  / re-exported in the prelude?

Reply:
Yep, good call, I added it to the prelude